java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -1331,11 +1331,11 @@ public void smove(BaseClient client) {
         String setKey3 = "{key}" + UUID.randomUUID();
         String nonSetKey = "{key}" + UUID.randomUUID();
 
-        assertEquals(3, client.sadd(setKey1, new String[] {"1", "2", "3"}).get());
```

Comment:
Add `GlideString` case, but not in the expense of `String`

Reply:
there are plenty of calls to `sadd` with strings and not GlideString in the integration tests